portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,41 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Elvis!</h1>");
-  }
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String message;
+         
+        ArrayList<String> greetings = new ArrayList<String>();
```

Comment:
The constructor's generic type argument can be inferred.

```java
ArrayList<String> greetings = new ArrayList<>();
```

https://docs.oracle.com/javase/tutorial/java/generics/genTypeInference.html